src/pages/Main/components/common/DirectionBtn.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import * as Common from './Common';
+
+const DirectionBtn = ({ index, setIndex, carousel }) => {
+  return (
+    <>
+      <button
+        onClick={() => {
+          Common.goToPrev(index, setIndex, carousel);
+        }}
+        className="prevBtn"
+      >
+        <i className="fa-solid fa-arrow-left" />
+      </button>
+      <button
+        onClick={() => {
+          Common.goToNext(index, setIndex, carousel);
+        }}
+        className="nextBtn"
+      >
+        <i className="fa-solid fa-arrow-right" />
+      </button>
+    </>
+  );
+};
```

Comment:
```suggestion
const DirectionBtn = ({ index, setIndex, carousel }) => {
  const goToPrev = () => {
    Common.goToPrev(index, setIndex, carousel);
  };

  const goToNext = () => {
    Common.goToNext(index, setIndex, carousel);
  };

  return (
    <>
      <button onClick={goToPrev} className="prevBtn">
        <i className="fa-solid fa-arrow-left" />
      </button>
      <button onClick={goToNext} className="nextBtn">
        <i className="fa-solid fa-arrow-right" />
      </button>
    </>
  );
};
```

이벤트 함수는 UI 에서 분리해주시는게 가독성과 유지보수에 좋습니다 :-)

Reply:
감사합니다. 수정했습니다!